minimalkv/fsspecstore.py

```diff
@@ -96,16 +99,27 @@ def __init__(
         ----------
         prefix: str, optional
             The prefix to use on the FSSpecStore when storing keys.
-        mkdir_prefix: Boolean
+        create_prefix: bool, optional
             If True, the prefix will be created if it does not exist.
             Analogous to the create_if_missing parameter in AzureBlockBlobStore or GoogleCloudStore.
         write_kwargs: dict, optional
             Additional keyword arguments to pass to the fsspec FileSystem when writing files.
+        custom_fs: AbstractFileSystem, optional
+            If given, use this fsspec FileSystem instead of creating a new one.
         """
         write_kwargs = write_kwargs or {}
-        self.prefix = prefix
-        self.mkdir_prefix = mkdir_prefix
+        self._prefix = prefix
+
+        if mkdir_prefix is not None:
+            warnings.warn(
+                "The mkdir_prefix parameter is deprecated and will be removed in a future version. "
+                "Use create_prefix instead.",
+                DeprecationWarning,
+            )
+        self._create_prefix = create_prefix or mkdir_prefix or True
```

Comment:
Why did you rename `mkdir_prefix` to `_create_prefix` instead of `_mkdir_prefix`?

Reply:
I didn't like the name, as it might suggest that it's a prefix that's used for `mkdir` or similar. But with the doc comment it shouldn't be too bad.